modin/pandas/test/dataframe/test_window.py

```diff
@@ -612,37 +610,29 @@ def test_quantile(request, data, q):
     modin_df = pd.DataFrame(data)
     pandas_df = pandas.DataFrame(data)
 
-    if not name_contains(request.node.name, no_numeric_dfs):
-        df_equals(modin_df.quantile(q), pandas_df.quantile(q))
-        df_equals(modin_df.quantile(q, axis=1), pandas_df.quantile(q, axis=1))
+    df_equals(modin_df.quantile(q), pandas_df.quantile(q))
+    df_equals(modin_df.quantile(q, axis=1), pandas_df.quantile(q, axis=1))
 
-        try:
-            pandas_result = pandas_df.quantile(q, axis=1, numeric_only=False)
-        except Exception as e:
-            with pytest.raises(type(e)):
-                modin_df.quantile(q, axis=1, numeric_only=False)
-        else:
-            modin_result = modin_df.quantile(q, axis=1, numeric_only=False)
-            df_equals(modin_result, pandas_result)
+    try:
```

Comment:
Are these changes just indentation changes? Shouldn't we remove the try except since pandas no longer raises an exception on non_numeric dtypes?